ingest/app.js

```diff
@@ -0,0 +1,32 @@
+const Ably = require('ably');
+const {PubSub} = require('@google-cloud/pubsub');
+const {SecretManagerServiceClient} = require('@google-cloud/secret-manager');
+
+const projectId = process.env.PROJECT;
+const pubSub = new PubSub({projectId});
+const secretManager = new SecretManagerServiceClient({projectId});
+
+async function main(
+  topicName = process.env.TOPIC,
+  subscriptionName = process.env.SUBSCRIPTION
+) {
+  const ablyApiKeySecretRef = `projects/${projectId}/secrets/ABLY_API_KEY/versions/latest`
+  const [ablyApiKeySecret] = await secretManager.accessSecretVersion({
```

Comment:
How is the API key secret set up - manually before this is run? If so, is it documented anywhere?
Otherwise this might catch us out if we ever rebuild from scratch.

Reply:
Currently the secret is set up manually. I'll add a note to the instructions on confluence and a README.md to this repo